lightning/src/chain/channelmonitor.rs

```diff
@@ -2299,6 +2321,39 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 	pub fn get_and_clear_pending_events(&mut self) -> Vec<Event> {
 		let mut ret = Vec::new();
 		mem::swap(&mut ret, &mut self.pending_events);
+		#[cfg(test)] // TODO: remove-on-anchors-release
+		for claim_event in self.onchain_tx_handler.get_and_clear_pending_claim_events().drain(..) {
+			match claim_event {
+				ClaimEvent::BumpCommitment {
+					package_target_feerate_sat_per_1000_weight, commitment_tx, anchor_output_idx,
+				} => {
+					let commitment_txid = commitment_tx.txid();
+					debug_assert_eq!(self.current_holder_commitment_tx.txid, commitment_txid);
+					let pending_htlcs = self.current_holder_commitment_tx.htlc_outputs.iter()
```

Comment:
this seems like something quite useful that perhaps ought to be defined in an easily callable function, either here or on a commitment_tx